### PR TITLE
Fix mad queue

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/14269

also fixes https://github.com/mozilla/addons-server/issues/14085#issuecomment-628108792

---

Not sure how I missed that given that the security queues are similar...

### CODE REVIEW

**Comment by willdurand** [comment]
@diox do you think that this fix should be cherry-picked?

**Comment by diox** [comment]
I'll review it properly later but it looks ok, I don't mind cherry-picking it.

File: src/olympia/reviewers/utils.py [hunk 1]
```diff
@@ -271,7 +271,33 @@ def render_addon_name(self, record):
 
 
 class MadReviewTable(AutoApprovedTable):
-    pass
+    def render_addon_name(self, record):
+        rval = [jinja2.escape(record.name)]
+        listed_versions = record.versions.filter(
+            channel=amo.RELEASE_CHANNEL_LISTED,
+            versionscannerflags__needs_human_review_by_mad=True,
```
[the remark this answers is not in the document]
Comment (willdurand) [reply src/olympia/reviewers/utils.py]: ha yes, makes sense, thanks. I updated the patch.

File: src/olympia/reviewers/utils.py [hunk 2]
```diff
@@ -241,37 +241,46 @@ def _get_addon_name_url(self, record):
...
             )
+
         return ''.join(rval)
 
 
-class MadReviewTable(AutoApprovedTable):
-    pass
+class MadReviewTable(ScannersReviewTable):
+    listed_text = _('Listed versions ({0})')
+    unlisted_text = _('Unlisted versions ({0})')
```
Comment (diox) [inline src/olympia/reviewers/utils.py:282]: Not sure this is better than the text from the parent class. Ultimately, we're only showing specific versions, not all, so without the extra `needing human review` bit it's a bit misleading, maybe.

Comment (willdurand) [reply src/olympia/reviewers/utils.py:282]: The name of the queue is "needs human review", that's why I removed it here.

**Pull request merged**